testing/acceptance_test.py

```diff
@@ -1066,6 +1066,29 @@ def test_worker_id1(worker_id, run_num):
         assert worker_ids == {"gw0", "gw1"}
 
 
+@pytest.mark.parametrize("n", [0, 2])
+def test_session_uuid_fixture(testdir, n):
```

Comment:
This test is quite similar to the one testing the `worker_id` fixture but it felt wrong to parametrize them.